lldb/source/Plugins/TypeSystem/Clang/TypeSystemClang.cpp

```diff
@@ -2771,6 +2771,9 @@ static bool GetCompleteQualType(clang::ASTContext *ast,
         ast, llvm::cast<clang::AttributedType>(qual_type)->getModifiedType(),
         allow_completion);
 
+  case clang::Type::MemberPointer:
+    return !qual_type.getTypePtr()->isIncompleteType();
```

Comment:
I see, so for non-MSVC ABI `isIncompleteType` will pretty much work as before, and trivially return true. Otherwise we check for the `MSInheritanceAttr`. Seems reasonable to me